tests/infra_test.go

```diff
@@ -1435,12 +1435,12 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", Serial, decorators.SigCo
 			})
 		})
 
-		Context("[Serial]node with obsolete host-model cpuModel", func() {
+		Context("[Serial]node with obsolete host-model cpuModel", Serial, func() {
 
 			expectSerialRun := func() {
-				suiteConfig, _ := GinkgoConfiguration()
-				Expect(suiteConfig.ParallelTotal).To(Equal(1), "this test is supported for serial tests only")
+				Expect(CurrentSpecReport().IsSerial).To(BeTrue(), "this test is supported for serial tests only")
```

Comment:
Since we are running all tests in this container as `Serial`, shouldn't we move this assertion into the `BeforeEach`?

Reply:
That function is used on in the event expectation. To check that the event we are expecting does not come from a parallel execution, we want to be sure we are not receiving spurious events.
https://github.com/kubevirt/kubevirt/pull/8848 This PR should help in this way.

Reply:
Ah, thanks for clarification about that. I was just thinking that since the container already has the `Serial` label, we could check it first in the BeforeEach. Otherwise I'd fear that this assertion would only be called for some tests. But I don't want to block on that.

Reply:
I would wait until the other PR is merged, and delete it directly :)